scripts/clean-java-files.ps1

```diff
@@ -1,8 +1,8 @@
-$extensionsAndGeneratedDirectories = Get-ChildItem $env:MainDirectory -Include extensions,generated -Recurse -Directory
+$modelsAndRequestsDirectories = Get-ChildItem $env:MainDirectory -Include models,requests -Recurse -Directory
 
-# this list should be updated if a new hand-crafted extension is added to one of the extensions/ directories
-$filesThatShouldNotBeDeleted = "UploadSession.java","DateOnly.java","TimeOfDay.java","Multipart.java","ChunkedUploadRequest.java","ChunkedUploadResult.java","CustomRequestBuilder.java"
-foreach ($directory in $extensionsAndGeneratedDirectories)
+# this list should be updated if a new hand-crafted extension is added to one of the directories
+$filesThatShouldNotBeDeleted = @()
```

Comment:
remove this parameter and the -Exclude flag below.

Reply:
```suggestion
```